### DISCUSSION TITLE
Help us choose a default voice!

### DISCUSSION THREAD

**Discussion by JarbasAl** [OpenVoiceOS/ovos-buildroot#101] [Q&A]
ovos-core can run fully offline and without backend, you might not even have a pairing step

This means we need a default value for voice that works offline!

We also need to have a fallback_tts value, this one is used when the main TTS fails. 

It's probably ok if we use the same TTS since the users will likely only change the main voice and this is the voice we always ensure is available

We will always need some binary installed, none of the local tts plugins are pure python, as long as it is FOSS and easily available we can consider any voice

At some point we might want to have a default per language, that will require code changes but we can start collecting data to inform that decision

Add more voices to the list below!
- [ ] Alan Pope - [Mimic](https://github.com/OpenVoiceOS/ovos-tts-plugin-mimic) - `"en-US"` - Male
- [ ] [PicoTTS](https://github.com/OpenVoiceOS/ovos-tts-plugin-pico) - `"de-DE", "es-ES", "fr-FR", "it-IT", "en-US"` - Female
- [ ] m1 - [espeak-ng](https://github.com/OpenVoiceOS/ovos-tts-plugin-espeakNG) - [all languages](https://github.com/espeak-ng/espeak-ng/blob/master/docs/languages.md) - Male
- [ ] f1 - [espeak-ng](https://github.com/OpenVoiceOS/ovos-tts-plugin-espeakNG) - [all languages](https://github.com/espeak-ng/espeak-ng/blob/master/docs/languages.md) - Female
- [ ] ...

**Comment by builderjer** [comment]
I like the voices from Larynx. 
https://github.com/NeonGeckoCom/neon-tts-plugin-larynx_server/tree/refactor/bump_opm

Especially if this works.

**Reply by ChanceNCounter** [reply]
https://rhasspy.github.io/larynx/#en-us

* ljspeech
* southern_english_female
* southern_english_male

all had, I thought, compelling samples.

**Comment by Akul2010** [comment]
The espeak male voices sound better than the female voices. They both are kind of robotic, though.

**Comment by kha84** [comment]
Hey! Sorry, I'm late to the party :) 

I just recently stumbled upon OpenVoiceOS and I really love the direction you're guys moving it into :heart:

As for the standard voice, some time ago, I was playing with different TTS models using Coqui API and I've made this simple script to run-them-all and generate a report to compare them. I was primarily looking for the ones, which are giving the best audio results, while keeping the least pressure on hardware. Have a look - https://github.com/kha84/tts-comparison

**Comment by JarbasAl** [comment]
I am adding existing plugins to this repo https://github.com/OpenVoiceOS/ovos-tts-auto-cache

all samples there are relevant

**Comment by andlo** [comment]
I like Allan Pope voice, as it sounds like Marvin the Paranoid Android from The Hitchhiker's Guide to the Galaxy :)

**Comment by erikjber** [comment]
I personally prefer `en_US/hifi-tts_low` speaker `92`. She sounds British and therefore classy.

**Reply by builderjer** [reply]
I use `en_US/vctk_low#s5 because of the same reason...and my wife likes it.  I will check that one also

**Comment by erikjber** [comment]


**Reply by JarbasAl** [reply]
that command is meant to launch skills standalone, for example in their own docker container or just for easy development. It's a new feature we just introduced

`:skills`  command has not been updated in a long time, ovos-cli-client is in maintenance mode and not getting active development, we just fix bugs as they are reported

we are diagnosing issues around the skill loading from folder, we got some more in our chat and are looking into it

**Reply by erikjber** [reply]
I just realised I posted this in the wrong thread. Apologies. I will be deleting it shortly. This is the correct thread: https://github.com/orgs/OpenVoiceOS/discussions/168
